Training_Center_Task_10/Tests.cs

```diff
@@ -0,0 +1,301 @@
+                }
+            }
+
+            //Refresh page
+            _driver.Navigate().Refresh();
+        }
+
+        [Description("Task 9")]
+        [TestMethod]
+        public void ReturnNamePositionOffice()
+        {
+            _driver.Navigate().GoToUrl("https://demo.seleniumeasy.com/table-sort-search-demo.html");
+
+            //Select "10" from dropdown
+            IWebElement showEntriesDropdown = _driver.FindElement(By.CssSelector("select[name='example_length']"), 10);
+            SelectElement selections = new SelectElement(showEntriesDropdown);
+            selections.SelectByValue("10");
+
+            //Create a loop which gets all row values and puts them in a list
+            List<EmployeeInfo> employees = new List<EmployeeInfo>();
+            FillInListOfEmployees(employees);
+
+            Assert.AreEqual("A. Cox", employees[0].Name);
+            Assert.AreEqual("B. Greer", employees[1].Name);
+            Assert.AreEqual("G. Joyce", employees[2].Name);
+            Assert.AreEqual("G. Winters", employees[3].Name);
+            Assert.AreEqual("H. Chandler", employees[4].Name);
+            Assert.AreEqual("M. Silva", employees[5].Name);
+            Assert.AreEqual("S. Burks", employees[6].Name);
+
+
+        }
+
+        public List<EmployeeInfo> FillInListOfEmployees(List<EmployeeInfo> employees)
+        {
+            int counter = 2;
+            while (true)
+            {
+                IWebElement table = _driver.FindElement(By.Id("example"));
+                var rows = table.FindElements(By.TagName("tr"));
+
+                foreach (var row in rows)
+                {
+                    List<string> userData = new List<string>();
+                    var data = row.FindElements(By.TagName("td"));
+                    if (data.Count != 0)
+                    {
+                        foreach (var cell in data)
+                        {
+                            userData.Add(cell.Text);
+                        }
+                    }
+                    else
+                    {
+                        continue;
+                    }
+
+                    int age = int.Parse(userData[3]);
+                    int salary = int.Parse(Regex.Replace(userData[5], "[^0-9]", ""));
+
+                    //Check whether employee is more > 39 years old and his salary is less than or equals $200,000
+                    //If both are true, add him to respective list.
+
+                    if (age > 39 && salary <= 200000)
+                    {
+                        EmployeeInfo employeeInfo = new EmployeeInfo();
+                        employeeInfo.Name = userData[0];
+                        employeeInfo.Position = userData[1];
+                        employeeInfo.Office = userData[2];
+                        employees.Add(employeeInfo);
+                    }
+                }
+
+                //Check whether there is next page and if yes - click the page number
```

Comment:
not a good way to build logic around try/catch

Reply:
Made new logic around Next button class attribute change